og_ui/og_ui.info.yml

```diff
@@ -7,5 +7,6 @@ core: 8.x
 
 dependencies:
   - og
+  - field_ui
```

Comment:
og_ui D7 had ` $url = array('!url' => l(t('Manage fields'), 'admin/structure/types/manage/' . str_replace('_', '-', $node_type) . '/fields'));` which basically created a soft dependency on Field UI. In D8 that's `$url = Link::createFromRoute(t('Manage fields'), FieldUI::getOverviewRouteInfo($entity_type_id, $bundle));` which is a hard dependency and that's why I added it.